### ISSUE TITLE
trained model perform different

### ISSUE THREAD

**Issue by Alexanderisgod** [OPEN-AIR-SUN/mars#56]
Hi, dear author,
i use the command "ns-train nsg-vkitti-car-depth-recon --data /data/datasets/VKITTI2/Scene06/clone "  to train a model。
however, the train results is different with yours 
![image](https://github.com/OPEN-AIR-SUN/mars/assets/41862202/8777c763-3070-4414-b5b9-34d00de9428d)

could you give me some advice, thanks a lot!

**Comment by xBeho1der** [comment]
Hi! Please provide more information, like config, experiment settings, wandb link, etc.

**Comment by Alexanderisgod** [comment]
> Hi! Please provide more information, like config, experiment settings, wandb link, etc.

thank you ,  i list some wandb figs and exp config as below.

![image](https://github.com/OPEN-AIR-SUN/mars/assets/41862202/4c39b942-88d3-4a35-a091-ea6d1de2d62e)
![image](https://github.com/OPEN-AIR-SUN/mars/assets/41862202/a4779d22-6965-4d3d-a301-6694b30e819e)
![image](https://github.com/OPEN-AIR-SUN/mars/assets/41862202/88ce703d-9794-47f7-8848-b231ae36b164)


VKITTI_Recon_NSG_Car_Depth = MethodSpecification(
    config=TrainerConfig(
        method_name="nsg-vkitti-car-depth-recon",
        steps_per_eval_image=STEPS_PER_EVAL_IMAGE,
        steps_per_eval_all_images=STEPS_PER_EVAL_ALL_IMAGES,
        steps_per_save=STEPS_PER_SAVE,
        save_only_latest_checkpoint=False,
        max_num_iterations=MAX_NUM_ITERATIONS,
        mixed_precision=False,
        use_grad_scaler=True,
        log_gradients=True,
        pipeline=NSGPipelineConfig(
            datamanager=NSGkittiDataManagerConfig(
                dataparser=NSGvkittiDataParserConfig(
                    use_car_latents=True,
                    use_depth=True,
                    car_object_latents_path=Path(
                        "/data/datasets/VKITTI2/car_nerfs/latents/latent_codes06.pt"
                    ),
                    split_setting="reconstruction",
                    car_nerf_state_dict_path=Path("/data/datasets/VKITTI2/car_nerfs/state_dict/epoch_805.ckpt"),
                ),
                train_num_rays_per_batch=4096,
                eval_num_rays_per_batch=4096,
                camera_optimizer=CameraOptimizerConfig(mode="off"),
            ),
            model=SceneGraphModelConfig(
                background_model=NerfactoModelConfig(),
                object_model_template=CarNeRFModelConfig(_target=CarNeRF),
                object_representation="class-wise",
                object_ray_sample_strategy="remove-bg",
            ),
        ),
        optimizers={
            "background_model": {
                "optimizer": RAdamOptimizerConfig(lr=1e-3, eps=1e-15),
                "scheduler": ExponentialDecaySchedulerConfig(lr_final=1e-5, max_steps=200000),
            },
            "object_model": {
                "optimizer": RAdamOptimizerConfig(lr=5e-3, eps=1e-15),
                "scheduler": ExponentialDecaySchedulerConfig(lr_final=1e-5, max_steps=200000),
            },
        },
        # viewer=ViewerConfig(num_rays_per_chunk=1 << 15),
        vis="wandb",
    ),
    description="Neural Scene Graph implementation with vanilla-NeRF model for backgruond and object models.",
)

**Comment by Alexanderisgod** [comment]
> Hi! Please provide more information, like config, experiment settings, wandb link, etc.

i found maybe i choose a  wrong pipeline, i use "nsg-vkitti-car-depth-nvs" and the result is aligned with yours.

**Comment by wuzirui** [comment]
great to hear that :)

**Comment by Alexanderisgod** [comment]
> great to hear that :)

sorry to bother you again, how can i implement the model for multi-GPU training or increasing batch size.

**Comment by wuzirui** [comment]
1. we have not explored multi-GPU training yet; I suggest referring the nerfstudio community
2. "increasing batch size" can be done by changing nerfstudio configs, e.g. train_num_rays_per_batch
